gui/util/range_slider.py

```diff
@@ -0,0 +1,294 @@
+"""
+Range slider, extended QWidget slider for napari-gui.
+"""
+
+from PyQt5 import QtCore, QtGui, QtWidgets
+
+
+class QRangeSlider(QtWidgets.QWidget):
+    """
+    QRangeSlider class, super class for QVRangeSlider and QHRangeSlider.
+    """
+
+    rangeChanged = QtCore.pyqtSignal(float, float)
```

Comment:
I remembered why that was the case. You need to write override slot for your signal if you make it an instance member. Otherwise it is not thread safe and easily crashes. Qt thing. That's why will leave it as a class attribute.

Reply:
Does this mean that when I have multiple sliders and subscribe to a range change event on one slider, I will be notified when the other is changed as well?